app/assets/sass/components/_header.scss

```diff
@@ -0,0 +1,144 @@
+.app-header {
```

Comment:
Do we still need all this?

Reply:
It appears the whole header component is no longer used. Happy for it to be removed?